src/test/java/org/folio/rs/service/ReturnItemServiceTest.java

```diff
@@ -0,0 +1,196 @@
+package org.folio.rs.service;
+
+import org.folio.rs.client.CirculationClient;
+import org.folio.rs.client.InventoryClient;
+import org.folio.rs.client.UsersClient;
+import org.folio.rs.domain.dto.CheckInItem;
+import org.folio.rs.domain.dto.Request;
+import org.folio.rs.domain.dto.ResultList;
+import org.folio.rs.domain.dto.User;
+import org.folio.rs.domain.entity.RetrievalQueueRecord;
+import org.folio.rs.domain.dto.Item;
+import org.folio.rs.error.ItemReturnException;
+import org.folio.rs.repository.RetrievalQueueRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Collections;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class ReturnItemServiceTest {
```

Comment:
Why we use two separate test classes for Return flow and returnItemService? Is it possible to have all of them in the one class just to correspond all other tests in the module?